inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -214,7 +214,11 @@ cdef class Blob:
 
     def __deepcopy__(self, memodict):
         res = Blob(deepcopy(self.tensor_desc, memodict), deepcopy(self._array_data, memodict))
-        res.buffer[:] = deepcopy(self.buffer[:], memodict)
+        precision = self.tensor_desc.precision
+        if precision in ["FP16","BF16"]:
```

Comment:
not sure that we should copy BF16 as float16

Reply:
Looks like for BF16 we still have a problem anyway